blocks/agent-communities/agent-communities.js

```diff
@@ -0,0 +1,100 @@
+import {
+  a, div,
+  img,
+  span,
+  p,
+} from '../../scripts/dom-helpers.js';
+
+export default async function decorate(block) {
+  block.classList.add('cards');
+  // const url = `/commonwealth-real-estate-ma312${window.location.pathname}`;
+  const url = '/commonwealth-real-estate-ma312/east-greenwich/barry-alofsin/cid-473521';
+  const bhhsCode = 'bhhs-ma312';
+
+  const finalUrl = new URL('https://api.liveby.com/v1/pages');
+  finalUrl.search = new URLSearchParams({
+    id: bhhsCode,
+    ref: url,
+  });
+
+  try {
+    const response1 = await fetch(finalUrl, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    const data1 = await response1.json();
+
+    const response2 = await fetch('https://api.liveby.com/v1/neighborhood-brokerage-lookup/pages', {
```

Comment:
These calls can't be part of the immediate decorate function execution. Since they will be to another domain, it will create experience issues with respect to the PSI score.

you'll need to set up the CSS to create an empty space that this block will fill, after a timeout (usually 3s)